cmsl1t/recalc/met.py

```diff
@@ -2,8 +2,27 @@
 import numpy as np
 
 
+class met(object):
+    def __init__(self, metx, mety):
+        self.metx = metx
+        self.mety = mety
+
+    def __getitem__(self, index):
```

Comment:
This was just to allow the existing unit tests to continue functioning. I'll remove this here and change the unit tests since I agree that it's less clear as an interface